src/chemistry/utils/tracer_data.F90

```diff
@@ -397,13 +397,13 @@ subroutine trcdata_init( specifier, filename, filelist, datapath, flds, file, &
 
        allocate( file%hyam(file%nlev),  file%hybm(file%nlev), stat=astat )
        if( astat /= 0 ) then
-          write(iulog,*) 'trcdata_init: file%hyam,file%hybm allocation error = ',astat
+          write(iulog,'(a,i8)') 'trcdata_init: file%hyam,file%hybm allocation error = ',astat
           call endrun('trcdata_init: failed to allocate file%hyam and file%hybm arrays')
```

Comment:
This is another case where using `int2str` (`use string_utils, only: int2str`) can save you having to create a separate `write` statement:
```suggestion
          call endrun('trcdata_init: failed to allocate file%hyam and file%hybm arrays, error code = '//int2str(astat))
```